saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -1238,11 +1254,26 @@ def make_order_line_calculations(
                 )
             )
 
+        prices_entered_with_tax = True
+        if channel := order_data.channel:
+            prices_entered_with_tax = channel.tax_configuration.prices_entered_with_tax
```

Comment:
Question:  Should we fetch it while creating `order_data`? `channel.tax_configuration.prices_entered_with_tax` could generate additional SQL queries per each line.

Reply:
@fowczarek you were right. I updated the code, to fetch the tax configurations in batches.